cornice_swagger/swagger.py

```diff
@@ -470,8 +485,12 @@ def generate_swagger_spec(services, title, version, **kwargs):
     """Utility to turn cornice web services into a Swagger-readable file.
     """
 
+    def get_tags_from_path(service):
+        return [service.path.split("/")[1]]
```

Comment:
> Can this error if the service path doesn't have a slash in it?

It actually can (I just tested it) but I've never seen an example without it. Also, this is just there for backward compatibility as well, but we should definitely add an IndexError exception here.